docs/pages/inboxes/history-sync.md

```diff
@@ -0,0 +1,154 @@
+code sample needed
+```
+
+```kotlin [Kotlin]
+code sample needed
+```
+
+```swift [Swift]
+code sample needed
+```
+
+:::
+
+### Sync messages
+
+Use this method to sync message history.
+
+:::code-group
+
+```jsx [Browser]
+code sample needed
+```
+
+```jsx [Node]
+code sample needed
+```
+
+```jsx [React Native]
+code sample needed
+```
+
+```kotlin [Kotlin]
+code sample needed
+```
+
+```swift [Swift]
+code sample needed
+```
+
+:::
+
+## How history sync works
+
+When your app initializes an XMTP client and a `historySyncUrl` client option is present, history sync automatically triggers an initial sync request, pulling all message history into the new app installation and merging it with the existing app installations in the sync group.
+
+Ongoing updates to history are streamed automatically. Updates, whether for user consent preferences or messages, are sent across the sync group, ensuring all app installations have up-to-date information.
+
+History syncs are accomplished using these components:
+
+- Sync group
+- Sync worker
+- History server
+
+### Sync group
+
+A sync group is a special [Messaging Layer Security](/protocol/specs) group that includes all of the user’s devices. The sync group is used to send serialized sync messages across the user's devices. The sync group is filtered out of most queries by default so they don’t appear in the user’s inbox.
+
+### Sync worker
+
+A sync worker is a spawned background worker that listens for sync events, and processes them accordingly. This worker:
+
+- Emits and updates consent
+- Creates and consumes archive payloads for and from other devices
+- Keeps preferences synced across devices
+
+### History server
+
+A history server acts as a bucket that holds encrypted sync payloads. The URL location of these payloads and the password (cipher encryption key) to decrypt these payloads is sent over the sync group for the recipient to decrypt.
+
+## FAQ
+
+### A user logged into a new app installation and doesn't see their conversations. What's going on?
+
+The history sync feature uses a [sync worker](#sync-worker) that runs every 30 minutes to add any new app installations to a user's existing conversations. It may take up to 30 minutes for all of a user's conversations to appear.
+
+You can give users a way to call sync conversations to enable them to get their conversations instantly from a pre-existing and online app installation.
+
+### A user logged into a new app installation and sees their conversations, but no messages. What's going on?
+
+Ensure that you've initiated a call to sync messages and that you've opened a pre-existing and online app installation to receive the sync and begin the syncing process.
+
+### I called a sync method (messages, consent state, or conversations), but nothing is happening. What's going on?
+
+After requesting a sync for one app installation, you must open a pre-existing and online app installation to receive the sync and begin the syncing process.
```

Comment:
Maye we should add another FAQ

- When should I call for a manual sync
A: If for some reason the automatic sync didn't work, and the user wants to force a new sync. Ideally this would be an option in the app's settings under sync with an option like "Force Sync Now", indicating that most of the time the library makes a best effort by itself to stay in sync automatically.